junitparser/junitparser.py

```diff
@@ -567,23 +567,32 @@ class Skipped(Result):
     _tag = "skipped"
 
     def __eq__(self, other):
-        return super(Skipped, self).__eq__(other)
+        if (isinstance(other, Skipped)):
```

Comment:
maybe change the `__eq__` function in the parent class to 
`return type(self) == type(other) and ...`
is more concise.

Also, make sure coverage doesn't drop, please.